etc/profile-a-l/joplin.profile

```diff
@@ -0,0 +1,71 @@
+# Firejail profile for joplin
+# Description: A note taking and to-do application
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include joplin.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+include allow-bin-sh.inc
+include allow-nodejs.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+include disable-xdg.inc
+
+include whitelist-common.inc
+
+mkdir ${HOME}/.config/joplin
+whitelist ${HOME}/.config/joplin
+
+whitelist ${HOME}/.config/nano
+read-only ${HOME}/.config/nano
+whitelist ${HOME}/.emacs
+read-only ${HOME}/.emacs
+whitelist ${HOME}/.emacs.d
+read-only ${HOME}/.emacs.d
+whitelist ${HOME}/.nanorc
+read-only ${HOME}/.nanorc
+whitelist ${HOME}/.vim
+read-only ${HOME}/.vim
+whitelist ${HOME}/.viminfo
+read-only ${HOME}/.viminfo
+whitelist ${HOME}/.vimrc
+read-only ${HOME}/.vimrc
+
+apparmor
+caps.drop all
+machine-id
+netfilter
+no3d
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+shell none
+
+dbus-user none
+dbus-system none
+
+private-dev
+private-cache
+private-etc none
```

Comment:
Does not work

Reply:
it is working. i used it from the [here](https://github.com/netblue30/firejail/blob/9adeaf0c6f302c3091496848bc6e3b732064e1e3/etc/profile-m-z/notify-send.profile#L53).

Reply:
ok i don't know what to add for starters i will add these thing you mentioned.